test/e2e/job.go

```diff
@@ -348,7 +346,7 @@ var _ = Describe("Job E2E Test", func() {
 					pri: masterPriority,
 					min: 1,
 					rep: 1,
-					req: twoCPU,
+					req: oneAndHalfCPU,
```

Comment:
This change has been made since, max node resource in KIND cluster is 1980m, so 2000m pod cannot be deployed in any of those nodes.  So it has been change to 1500m.